examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_proxy_operator.py

```diff
@@ -90,9 +90,12 @@ def launch_runs_for_task(self, context: Context, dag_id: str, task_id: str) -> N
             json={"query": ASSET_NODES_QUERY},
             timeout=3,
         )
+        logger.info(f"Fetching asset nodes for {dag_id}/{task_id}")
         asset_nodes_data = self.get_valid_graphql_response(response, "assetNodes")
+        logger.info(f"Got response {asset_nodes_data}")
         for asset_node in asset_nodes_data:
-            if matched_dag_id_task_id(asset_node, dag_id, task_id):
+            # jobs can be empty if it is an external asset (e.g. an automapped task)
```

Comment:
@dpeng817 flagging this business logic change from the last time you looked

Reply:
Looks reasonable.